docs/jupyter/NLTK.ipynb

```diff
@@ -0,0 +1,874 @@
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>Unnamed: 0</th>\n",
+       "      <th>created_at</th>\n",
+       "      <th>id</th>\n",
+       "      <th>author_id</th>\n",
+       "      <th>text</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>0</td>\n",
+       "      <td>2022-05-16T21:24:35.000Z</td>\n",
+       "      <td>1526312680226799618</td>\n",
+       "      <td>813286</td>\n",
+       "      <td>It’s despicable, it’s dangerous — and it needs...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>1</td>\n",
+       "      <td>2022-05-16T21:24:34.000Z</td>\n",
+       "      <td>1526312678951641088</td>\n",
+       "      <td>813286</td>\n",
+       "      <td>We need to repudiate in the strongest terms th...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>2</td>\n",
+       "      <td>2022-05-16T21:24:34.000Z</td>\n",
+       "      <td>1526312677521428480</td>\n",
+       "      <td>813286</td>\n",
+       "      <td>This weekend’s shootings in Buffalo offer a tr...</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   Unnamed: 0                created_at                   id  author_id  \\\n",
+       "0           0  2022-05-16T21:24:35.000Z  1526312680226799618     813286   \n",
+       "1           1  2022-05-16T21:24:34.000Z  1526312678951641088     813286   \n",
+       "2           2  2022-05-16T21:24:34.000Z  1526312677521428480     813286   \n",
+       "\n",
+       "                                                text  \n",
+       "0  It’s despicable, it’s dangerous — and it needs...  \n",
+       "1  We need to repudiate in the strongest terms th...  \n",
+       "2  This weekend’s shootings in Buffalo offer a tr...  "
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Import some Tweets from Barack Obama \n",
+    "df = pd.read_csv(\"https://raw.githubusercontent.com/kirenz/twitter-tweepy/main/tweets-obama.csv\")\n",
+    "df.head(3)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import some Tweets from Barack Obama \n",
+    "df_pandas = pandas.read_csv(\"https://raw.githubusercontent.com/kirenz/twitter-tweepy/main/tweets-obama.csv\")"
```

Comment:
```suggestion
    "pandas_df = pandas.read_csv(\"https://raw.githubusercontent.com/kirenz/twitter-tweepy/main/tweets-obama.csv\")"
```
fix everywhere where necessary.

Also, why are you playing around with Modin df but not playing with the pandas_df?

Reply:
Fixed! I had only replicated the visualization workflows in pandas to ensure the behavior is the same for that and modin (since even though the library API does not result in an error with Modin, it may not produce the desired visualization). But I've now added this for NLTK, statsmodels, tensorflow as those had APIs not interoperable with Modin/ would be interesting to see possible differences. I can do the same for HuggingFace, sklearn, xgboost but I'm not sure if seeing the workflows there with pandas would be valuable.